package.json

```diff
@@ -34,6 +34,7 @@
   },
   "homepage": "https://github.com/tipsi/tipsi-ui-kit#readme",
   "dependencies": {
+    "react-native-dash": "0.0.3",
```

Comment:
I believe we should move this into our project instead of relying on component without tests, especially because it's quite easy and shouldn't take too much time to support